testing/coercion.ts

```diff
@@ -38,21 +39,25 @@ function coerceParamValue(paramDef: ParamDef<any>, paramValue: any): any {
   if (isArrayType(paramDef.type)) {
     const valuesString = paramValue as string;
     const value = valuesString.length ? valuesString.split(',') : [];
-    return value.map(item => coerceParam((paramDef.type as ArrayType<Type>).items, item.trim()));
+    return value.map(item => coerceParam((paramDef.type as ArrayType<ParamType>).items, paramDef.name, item.trim()));
   }
-  return coerceParam(paramDef.type, paramValue);
+  return coerceParam(paramDef.type, paramDef.name, paramValue);
 }
 
-function coerceParam(type: Type, value: any): any {
+function coerceParam(type: ParamType, name: string, value: any): any {
   switch (type) {
     case Type.boolean:
+      if((value || '').toLowerCase() !== 'true' && (value || '').toLowerCase() !== 'false') {
```

Comment:
Nit, since we do it three times, let's just make a local var of `(value || '').toLowerCase()`